scripts/install-contracts/index.js

```diff
@@ -0,0 +1,322 @@
+const fs = require('fs');
+const {
+  CasperServiceByJsonRPC,
+  Keys,
+  Contracts,
+  CasperClient,
+  CLValueBuilder,
+  RuntimeArgs,
+} = require('casper-js-sdk');
+const path = require('path');
+const {
+  installContract,
+  waitForDeploy,
+  parseExecutionResult,
+  parseWriteContract,
+  stringToCLKey,
+} = require('./utils');
+
+const rawConfig = fs.readFileSync(path.resolve(__dirname, './config.json'), 'utf-8');
+
+const config = JSON.parse(rawConfig);
+
+const rpcAPI = new CasperServiceByJsonRPC(config.node_rpc_uri);
+const contractClient = new Contracts.Contract(new CasperClient(config.node_rpc_uri));
+
+async function main() {
+  const status = await rpcAPI.getStatus();
+
+  const pk = Keys.Ed25519.loadKeyPairFromPrivateFile(config.private_key_path);
+
+  console.log(`
+    Info: Installing contracts:
+      1) VariableRepositoryContract
+      2) ReputationContract
+      3) DaoIdsContract
+      4) VaNftContract
+      5) KycNftContract
+  `);
+
+  let contractsConfig = [
+    {
+      name: 'VariableRepositoryContract',
+      wasmPath: path.resolve(__dirname, config.contracts.VariableRepositoryContract.wasm_relative_path),
```

Comment:
Why not traferse the config and resolve paths along the way? With the current approach if a new contract is added to the config, it still has to be added here in the code.